Class/Common/Enchants/EnchantStatic.cpp

```diff
@@ -199,6 +202,9 @@ EnchantStatic::~EnchantStatic() {
     case EnchantName::EnchantWeaponSpellPower:
         pchar->get_stats()->decrease_base_spell_damage(30);
         break;
+    case EnchantName::EnchantBracerGreaterIntellect:
+        pchar->get_stats()->increase_intellect(7);
```

Comment:
This needs to be decrease_intellect(), as it currently keeps stacking intellect bonus when applying enchant and removing enchant.

Reply:
Fixed